qemu/tests/cfg/block_discard.cfg

```diff
@@ -22,11 +22,11 @@
     variants:
         - xfs:
             no RHEL.3 RHEL.4 RHEL.5 RHEL.6
-            create_partition_cmd = "parted -s DISK mklabel msdos && "
-            create_partition_cmd += "parted -s DISK mkpart primary xfs 2048s 100%"
+            create_disklabel_cmd = "parted -s DISK mklabel msdos"
```

Comment:
What do you think only change "parted -s DISK mklabel msdos &&" to
"parted -s DISK mklabel msdos && sleep 1 &&"
 it does not need other parts to be changed.

Reply:
good suggestion :)

Reply:
+1, thanks, done.